README.md

```diff
@@ -5,90 +5,33 @@ More info on the aws (s3/dynamo) backend supported by this module is found here:
 
-## Module options
-
-Options and configuration for this module are exposed via terraform variables.
-
-
-#### `backend_bucket`
-
-This is the only variable which has no default but is required. You will need to define this value in your terraform-aws-backend module block. There are a few ways to do this, here's a couple:
-
-```hcl
-module "backend" {
-  source = "github.com/samstav/terraform-aws-backend"
-  backend_bucket = "terraform-state-bucket"
-}
-```
-
-OR
-
-```hcl
-variable "backend_bucket" {
-  default = "terraform-state-bucket"
-}
-
-module "backend" {
-  source = "github.com/samstav/terraform-aws-backend"
-  backend_bucket = "${var.backend_bucket}"
-}
-```
-
-#### `dynamodb_lock_table_enabled`
-
-_Defaults to `true`._
-
-- Set to false or 0 to prevent this module from creating the DynamoDB table to use for terraform state locking and consistency. More info on locking for aws/s3 backends: https://www.terraform.io/docs/backends/types/s3.html. More information about how terraform handles booleans here: https://www.terraform.io/docs/configuration/variables.html"
-}
-
-#### `dynamodb_lock_table_stream_enabled`
-
-_Defaults to `false`._
-
-Affects terraform-aws-backend module behavior. Set to false or 0 to disable DynamoDB Streams for the table. More info on DynamoDB streams: https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Streams.html. More information about how terraform handles booleans here: https://www.terraform.io/docs/configuration/variables.html
-
-
-#### `dynamodb_lock_table_stream_view_type`
-
-_Defaults to `NEW_AND_OLD_IMAGES`_
-
-Only applies if `dynamodb_lock_table_stream_enabled` is true.
-
-#### `dynamodb_lock_table_name`
-
-_Defaults to `terraform-lock`_
-
-The name of your [terraform state locking](https://www.terraform.io/docs/state/locking.html) DynamoDB Table.
-
-#### `lock_table_read_capacity`
-
-_Defaults to `1` Read Capacity Unit._
-
-More on DynamoDB Capacity Units: https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/CapacityUnitCalculations.html
 
+# Bootstrapping your project
 
-#### `lock_table_write_capacity`
-_Defaults to `1` Write Capacity Unit._
+This terraform module helps you bootstrap any project which uses terraform for infrastructure management. [This module has a few options which are documented below. They allow you to change the behavior of this module.](#module-options)
 
-More on DynamoDB Capacity Units: https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/CapacityUnitCalculations.html
+**_Why does this exist?_**
 
+One of the most popular backend options for terraform is AWS (S3 for state, and DynamoDB for the lock table). Terraform requires the existence of an S3 bucket in which to store _state_ information about your project, and a DynamoDB table to use for locking (this prevents you, your collaborators, and CI from stepping on each other with terraform commands which either modify your state or the infrastructure itself).
```

Comment:
This should be clarified-- terraform only requires the existence of those things if you configure your project to require them, i.e. tell tf to use the aws backend.